forch/forch_proxy.py

```diff
@@ -0,0 +1,71 @@
+"""Module for proxy server to aggregate and serve data"""
+
+import functools
+import logging
+import threading
+import requests
+
+import forch.http_server
+
+LOGGER = logging.getLogger('proxy')
+DEFAULT_PROXY_PORT = 8080
+LOCALHOST = '0.0.0.0'
+
+
+class ForchProxy():
+    """Class that implements the module that creates a proxy server"""
+
+    def __init__(self, proxy_config):
+        self._proxy_config = proxy_config
+        self._proxy_port = self._proxy_config.proxy_port or DEFAULT_PROXY_PORT
+        self._pages = {}
+        self._proxy_server = None
+
+    def start(self):
+        """Start serving varz"""
```

Comment:
this class should just be a generic web proxy, so there should be no mention of varz in this file...

Reply:
Addressed.